dotnet/Microsoft.NET.Sdk.iOS/WorkloadManifest.targets

```diff
@@ -1,4 +1,7 @@
 <Project>
 	<Import Project="Sdk.props" Sdk="Microsoft.iOS.Sdk" Condition="'$(TargetPlatformIdentifier)' == 'iOS'" />
 	<Import Project="Sdk.props" Sdk="Microsoft.iOS.Windows.Sdk" Condition=" '$(TargetPlatformIdentifier)' == 'iOS' and $([MSBuild]::IsOSPlatform('windows'))" />
+
+	<Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm" Condition="'$(TargetPlatformIdentifier)' == 'iOS' And '$(RuntimeIdentifier)' == 'ios-arm'" />
```

Comment:
You should be able to extend the workloads defined in the Mono and not need to have these in your targets.  

`microsoft-net-runtime-ios` and `microsoft-net-runtime-tvos` are defined in https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in#L33-L63

Reply:
Those only work if `RunAOTCompilation` is set to true:

https://github.com/dotnet/runtime/blob/7e651850f7eeb69fa5e9a291197f22a9721a48a1/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets#L13-L18

and we can't set it to true in our WorkloadManifest.targets, because we might be loaded after you, which means that users would have to set it in their csproj, which we don't really want.

Reply:
In any case, I do believe the end goal would be to extend your workloads, but for P5 this solution seems to work. It's not necessarily what we'd end up doing in `main`.

Also, your workload is missing an AOT compiler for Mac Catalyst.

Reply:
> Also, your workload is missing an AOT compiler for Mac Catalyst.

I filed https://github.com/dotnet/runtime/issues/53579 for this.